storagenode/trust/service.go

```diff
@@ -93,29 +78,15 @@ func (pool *Pool) GetSignee(ctx context.Context, id storj.NodeID) (_ signing.Sig
 	info, ok := pool.trustedSatellites[id]
 	pool.mu.RUnlock()
 
-	if pool.trustAllSatellites {
-		// add a new entry
-		if !ok {
-			pool.mu.Lock()
-			// did another goroutine manage to make it first?
-			info, ok = pool.trustedSatellites[id]
-			if !ok {
-				info = &satelliteInfoCache{}
-				pool.trustedSatellites[id] = info
-			}
-			pool.mu.Unlock()
-		}
-	} else {
-		if !ok {
-			return nil, fmt.Errorf("signee %q is untrusted", id)
-		}
+	if !ok {
+		return nil, fmt.Errorf("signee %q is untrusted", id)
```

Comment:
should we have a class for this error? (it's probably to do it in another PR)

Reply:
agree, I think this could be its own err class vs fmt. but I think that would be fine to just add at the top of this file in this pr like `var ErrUntrusted = errs.Class("signee is untrusted:")` so then you could do `return nil, ErrUntrusted.New(id.String())`. but also I don't think this is a critical change.

Reply:
I currently made it use `Error` directly, since we don't need the distinction at the moment, I didn't add a new class. When we do, we can add more.